### ISSUE TITLE
API

### ISSUE THREAD

**Issue by ZiGaMi** [GeneralEmbeddedCLibraries/led#1]
API definition
FILE: led.h

```C

////////////////////////////////////////////////////////////////////////////////
// Functions
////////////////////////////////////////////////////////////////////////////////
led_status_t led_init   	(void);
led_status_t led_is_init        (bool * const p_is_init)
led_status_t led_hndl		(void);
led_status_t led_set    	(const led_num_t num, const led_state_t state);
led_status_t led_set_smooth	(const led_num_t num, const led_state_t state);
led_status_t led_blink		(const led_num_t num, const float32_t on_time, const float32_t period, const led_blink_t blink);
led_status_t led_blink_smooth	(const led_num_t num, const float32_t on_time, const float32_t period, const led_blink_t blink);

/**
 *  LED status
 */
typedef enum
{
	eLED_OK = 0,
        eLED_ERROR = 0x01,
} led_status_t;

/**
 *  Blink mode
 */
typedef enum
{
	eLED_BLINK_1X = 0,
	eLED_BLINK_2X,
	eLED_BLINK_3X,
	eLED_BLINK_4X,
	eLED_BLINK_5X,
	eLED_BLINK_CONTINUOUS,

	eLED_BLINK_NUM_OF
} led_blink_t;


```

**Comment by ZiGaMi** [comment]
Add configurations of LEDS:

FILE: led_cfg.h
```C

/**
 *  LED enumerations
 */
typedef enum
{
    eLED_L = 0,	/**<Left LED */
    eLED_R,		/**<Right LED */

    eLED_NUM_OF
} led_num_t;

```


FILE: led_cfg.c
```C

/**
 *  LED configurations table
 */
....

```

**Comment by ZiGaMi** [comment]
Solved in release V1.0.0
